Realm/Realm/Schema/RealmSchema.cs

```diff
@@ -213,7 +213,7 @@ public RealmSchema Build()
                 if (Count == 0)
                 {
                     throw new InvalidOperationException(
-                        "No RealmObjects. Has linker stripped them? See https://realm.io/docs/xamarin/latest/#linker-stripped-schema");
+                        "No RealmObjects. Has linker stripped them? See https://thereIsntSuchDocYet");
```

Comment:
Feel free to link to the legacy docs and open a ticket for the team to move the information to the new docs.